src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
Avoid using one character parameter names, instead use more descriptive names like 'person' etc.

Read more about it here: [JS Google Style Guide | Parameter names](https://google.github.io/styleguide/jsguide.html#naming-parameter-names)